index.html

```diff
@@ -65,5 +86,193 @@ <h2 class="SupportingText">I'am a Software Developer</h2>
+         <li> <button class="work2Button" type="button">HTML</button> </li>
+         <li> <button class="work2Button" type="button">Bootstrap</button> </li>
+         <li> <button class="work2Button" type="button">Ruby</button> </li>
+        </ul>
+        <button class="work2Link">See project</button>
+      </div>
+      <div class="work2">
+        <h3 class="work2title">
+          Profesional Art Printing Data
+        </h3>
+        <p class="work2Description">
+          A daily selection of privately personalized reads; no accounts or 
+          sign-ups required. Has been the industry's standard.
+        </p>
+        <ul class="work2Languages">
+         <li> <button class="work2Button" type="button">HTML</button> </li>
+         <li> <button class="work2Button" type="button">Bootstrap</button> </li>
+         <li> <button class="work2Button" type="button">Ruby</button> </li>
+        </ul>
+        <button class="work2Link">See project</button>
+      </div>
+      <div class="work2">
+        <h3 class="work2title">
+          Profesional Art Printing Data
+        </h3>
+        <p class="work2Description">
+          A daily selection of privately personalized reads; no accounts or 
+          sign-ups required. Has been the industry's standard.
+        </p>
+        <ul class="work2Languages">
+         <li> <button class="work2Button" type="button">HTML</button> </li>
+         <li> <button class="work2Button" type="button">Bootstrap</button> </li>
+         <li> <button class="work2Button" type="button">Ruby</button> </li>
+        </ul>
+        <button class="work2Link">See project</button>
+      </div>
+      
+    </div>
+  </div>
+</section>
+<!---About Me section-->
+
+<section class="Aboutmesession">
+ <div class="aboutMe">
+   <h2 class="aboutmeTitle">
+    About me
+  </h2>
+  <p class="aboutMeDescription">
+    Hello I’m a software developer! I can help you build a product , feature or 
+    website Look through some of my work and experience! If you like what you 
+    see and have a project you need coded, don’t hesitate to contact me.
+  </p>
+  <div class="resumeContainer">
+    <button class="resumebutton">Get my resume</button>
+  </div>
+  <div id="background-img2" class="background-img2"></div>
+  <img src="images/Vector 4 (1).png" alt="line" class="line2">
+ </div>
+ <div class="Language">
+  <h4>Language</h4>
+ </div>
+ 
+ <div class="Rectangle">
+  <a href="#"><img src="images/Rectangle 56.png" alt="Rectangle"></a>
+ </div>
+
+ <ul class="list1">
+  <li class="Ruby">JavaScript</li>
+  <li class="Ruby">Ruby</li>
+  <li class="Ruby">HTML</li>
+  <li class="Ruby">CSS</li>
+ </ul>
+
+ <div class="Language">
+  <h4>Frameworks</h4>
+ </div>
+
+ <div class="Rectangle55">
+  <a href="#"><img src="images/Rectangle 55.png" alt="Rectangle"></a>
+ </div>
+
+ <ul class="list1">
+  <li class="Ruby">React.js</li>
+  <li class="Ruby">Ruby on Rails</li>
+  <li class="Ruby">Rspect</li>
```

Comment:
- [ ] Please make sure the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, and each of them in a` li` tag as mentioned in the  [project requirement ](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#project--requirements)

> Please use `ul `and `li` for categories in "About myself" section.` Languages`, `Frameworks`, and `Skills` should be list items.

 Example snippet:

```sh

<ul>
<li>Languages</li>
<li>Frameworks</li>
<li>Skills</li>
</ul>

```